src/pybert/solvers/simbeor/__init__.py

```diff
@@ -31,9 +35,6 @@
 class Solver(slvr.Solver):
     """Simbeor-specific override of `pybert.solver.Solver` class."""
 
-    def __init__(self):
-        super().__init__()
-
```

Comment:
Superclass initialization no longer needs to be explicitly invoked?

Reply:
Correct! That was actually python 2 to 3 change!  As long as we aren't needing to pass anything, no need to explicitly do it!

https://docs.python.org/3/library/functions.html#super